doc/users/dflt_style_changes.rst

```diff
@@ -1065,6 +1065,17 @@ Z-order
 
 
 
+``ScalarFormatter`` tick label formatting with offsets
+======================================================
+
+With the default of ``rcParams['axes.formatter.useoffset'] = True``,
+an offset will be used when it will save 4 or more digits.  This can
```

Comment:
Maybe instead of "save", we can use "reduce"? Something like "... if it will reduce the (something) of the ticks by 4 or more digits." and then we just need to determine the correct "(something)"